README.md

```diff
@@ -104,18 +93,70 @@ The lighthouse report results are automatically printed to the **Deploy log** in
 2:35:17 PM: }
 ```
 
-We currently support the following settings, which are passed directly to Lighthouse:
+## Lighthouse plugin configuration options
+
+### Run Lighthouse for different site paths
+
+To understand how your site's performance compares across different parts of your site, you can specify different site paths where you'd like to run Lighthouse audits.  Updates to `netlify.toml` will take effect for new builds.
+
+For example, this `netlify.toml` file configures Lighthouse audits for the root path of a site and the contact page:
+
+```
+[[plugins]]
+  package = "@netlify/plugin-lighthouse"
+
+  # Generate a Lighthouse report for the site's root path
+  [[plugins.inputs.audits]]
+  path = ""
+
+  # Generate a Lighthouse report for the contact site path
+  [[plugins.inputs.audits]]
+  path = "contact"
+
+```
+
+### Run Lighthouse for the desktop experience
+
+By default, Lighthouse takes a mobile-first performance testing approach and runs its audits for the mobile device experience.  Updates to `netlify.toml` will take effect for new builds.
+
+You can optionally run Lighthouse audits for the desktop experience by including `preset = "desktop"` in your `netlify.toml` file:
```

Comment:
@jackbrewer I'm also not sure if this is "too much help." Let me know if there's a preferred README.md style here.